src/cli/commands/version.js

```diff
@@ -146,14 +144,16 @@ export async function setVersion(
       const flag = sign ? '-sm' : '-am';
       const prefix: string = String(config.getOption('version-tag-prefix'));
 
+      const gitRoot = (await spawnGit(['rev-parse', '--show-toplevel'])).trim();
+
       // add manifest
-      await spawn('git', ['add', pkgLoc]);
+      await spawnGit(['add', path.relative(gitRoot, pkgLoc)]);
```

Comment:
I think you need to use the `{cwd: gitRoot}` option, or it won't work if you're in a deep directory.